tests/acceptance/guid-node/registrations-test.ts

```diff
@@ -57,10 +57,157 @@ module('Acceptance | guid-node/registrations', hooks => {
+
+        assert.dom('[data-test-node-card-body]').includesText(registrationTitle);
+
+        await untrackedClick('[data-test-registrations-container] a[href="#drafts"]');
+
+        assert.dom('[data-test-registrations-pane]').isNotVisible();
+        assert.dom('[data-test-draft-registrations-pane]').isVisible();
+
+        assert.dom('[data-test-draft-registrations-pane]').includesText(
+            'There are no draft registrations of this project.',
+        );
+    });
+
+    test('logged in write-permission, 1 draft registration', async assert => {
+        const initiator = server.create('user', 'loggedIn');
+
+        const node = server.create('node', {
+            id: 'decaf',
+            currentUserPermissions: [Permission.Write, Permission.Read],
+        });
+
+        server.loadFixtures('schema-blocks');
+        server.loadFixtures('registration-schemas');
+        server.loadFixtures('registration-providers');
+        server.loadFixtures('licenses');
+
+        const registrationSchemaName = 'Prereg Challenge';
+        const registrationSchema = server.schema.registrationSchemas.findBy({ name: registrationSchemaName });
+
+        const registrationMetadata = {
+            q1: { comments: [], value: 'Registration Title', extra: [] },
+        };
+
+        draftRegisterNode(server, node, { initiator, registrationSchema, registrationMetadata });
+
+        const url = `/${node.id}/registrations`;
+
+        await visit(url);
+
+        assert.equal(currentURL(), url, `We are on ${url}`);
+
+        assert.dom('[data-test-new-registration-button]').doesNotExist(
+            'write-contributors cannot create new draft registration',
+        );
+
+        assert.dom('[data-test-registrations-pane]').includesText(
+            'There have been no completed registrations of this project.',
+        );
+
+        await untrackedClick('[data-test-registrations-container] a[href="#drafts"]');
+
+        assert.dom('[data-test-draft-registrations-pane]').isVisible();
+
+        assert.dom('[data-test-draft-registrations-pane]').doesNotIncludeText(
+            'There are no draft registrations of this project.',
+        );
+
+        assert.dom('[data-test-draft-registration-card]').exists({ count: 1 });
+
+        assert.dom('[data-test-draft-registration-card-title]').includesText(
+            'Prereg Challenge',
+        );
+
+        assert.dom('[data-test-draft-registration-card-progress-bar]').exists({ count: 1 });
+
+        assert.dom('[data-test-draft-registration-card-progress-bar]').isVisible();
+    });
+
     test('logged in admin, no registrations', async assert => {
         server.create('user', 'loggedIn');
 
-        const node = server.create('node', { id: 'decaf', currentUserPermissions: [Permission.Admin] });
+        const node = server.create('node', {
+            id: 'decaf',
+            currentUserPermissions: [Permission.Admin, Permission.Write, Permission.Read],
```

Comment:
could use the `currentUserAdmin` mirage trait instead (ditto for each time you specify all three permissions)